en/docs/learn/enterprise-integration-patterns/message-construction/event-message.md

```diff
@@ -13,89 +13,167 @@ The Event Message EIP is used for reliable, asynchronous event notification betw
 
-    1. Start the ESB profile and log into its Management Console. For instructions, see Starting the ESB profile in WSO2 EI Documentation.
+7. Set up and Start [ActiveMQ]({{base_path}}/install-and-setup/setup/brokers/configure-with-activemq).
 
-   2. Select the **Topics** menu from the **Main** menu, and then select the **Add** sub menu.
+!!!note
+    Make sure to configure the relevant [JMS parameters]({{base_path}}/reference/synapse-properties/transport-parameters/jms-transport-parameters) in the `deployment.toml` file.
+    ```
+    [[transport.jms.listener]]
+    name = "myTopicConnectionFactory"
+    parameter.initial_naming_factory = "org.apache.activemq.jndi.ActiveMQInitialContextFactory"
+    parameter.provider_url = "tcp://localhost:61616"
+    parameter.connection_factory_name = "TopicConnectionFactory"
+    parameter.connection_factory_type = "topic"
 
-   3. Enter the name `stockquote` for the topic and then click **Add Topic**.
+    [[transport.jms.sender]]
+    name = "myTopicSender"
+    parameter.initial_naming_factory = "org.apache.activemq.jndi.ActiveMQInitialContextFactory"
+    parameter.provider_url = "tcp://localhost:61616"
+    parameter.connection_factory_name = "TopicConnectionFactory"
+    parameter.connection_factory_type = "topic"
+    ```
 
-   4. In the Topic Browser tree, click the newly created `stockquote` topic and then click **Subscribe** to create a static subscription.
 
-   5. Enter the value http://localhost:9000/services/SimpleStockQuoteService in the **Event Sink URL** field and click **Subscribe**.
+## Execute the sample
 
-   6. Repeat these steps to add another subscriber in port `9001`.
+Send the following request to the service using SoapUI (or any other SOAP client).
 
-## Execute the sample
+```xml
+POST http://localhost:8290/services/EventGeneratorProxy
 
-Send the following message from a SOAP client like SoapUI to the ESB profile.
+Content-Type: text/xml;charset=UTF-8
+SOAPAction: "urn:getQuote"
 
-```
 <soapenv:Envelope xmlns:soapenv="http://schemas.xmlsoap.org/soap/envelope/" xmlns:ser="http://services.samples" xmlns:xsd="http://services.samples/xsd">
-   <soapenv:Header/>
-   <soapenv:Body>
-      <ser:getQuote>   
-         <ser:request>         
-            <xsd:symbol>foo</xsd:symbol>
-         </ser:request>        
-      </ser:getQuote>
-   </soapenv:Body>
+<soapenv:Body>
+    <ser:getQuote>
+        <ser:request>
+            <xsd:symbol>IBM</xsd:symbol>
+        </ser:request>
+    </ser:getQuote>
+</soapenv:Body>
 </soapenv:Envelope>
 ```
 
-Observe the two Axis2 server instances. Both instances will receive the request, which was sent from the client.
-
-Output of both Axis Server consoles:
+## Analyze the output
 
-samples.services.SimpleStockQuoteService :: Generating quote for : foo
+When you execute the command above, the request is sent to the StockQuoteProxy. Notice the following processed server log in WSO2 MI output:
```

Comment:
We have not defined a `StockQuoteProxy` in the example. Are you referring to the JMS topic here?

Reply:
Updated